templates/boot/generic-qemu-boot-template.jinja2

```diff
@@ -19,6 +22,11 @@
 {% endblock %}
 {% block actions %}
 
+{% if arch == 'riscv' %}
+tags:
+    - qemu-riscv64
```

Comment:
This is breaking things:
```
￼ERROR /data/workspace/bot.staging.kernelci.org/test-runner/workspace/test-runner/jobs/lab-baylibre/kernelci-staging.kernelci.org-staging-20200902.2-riscv-rv32_defconfig-gcc-8-no-dtb-qemu_riscv64-baseline_qemu.yaml: <Fault 400: "Invalid job definition: Device tag 'qemu-riscv64' does not exist in the database..">
```
I don't think using tags is going to work.  We should instead just be filtering which labs get to run RISC-V jobs on QEMU.